invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -256,16 +257,24 @@ <h2 id="files-heading">{{ _('Files') }}</h2>
                               <p>{{ _("Reason") }}: {{ record.access.embargo.reason }}</p>
                             {% endif %}
 
-                            {%- if user_request_enabled or guest_request_enabled  %}
+                            {%- if allow_user_requests or allow_guest_requests %}
                               <hr style="border-color: darkred;">
                               <p>
                                 <p>
-                                  <strong>Request access</strong>
+                                  <strong>{{ _("Request access") }}</strong>
                                 </p>
                                 <p>
-                                  <!-- TODO: Once implemented in the backend display the access description provided nad use this as fallback -->
-                                  If you would like to request access to these files, please fill out the form below.
+                                  {{ _("If you would like to request access to these files, please fill out the form below.") }}
                                 </p>
+                                <div>
+                                  {%- if record.parent.access.settings %}{%- set accept_conditions_text = record.parent.access.settings.accept_conditions_text %}{%- endif %}
+                                  {%- if accept_conditions_text %}
+                                    <p>
+                                      <h5>{{ _("You need to satisfy these conditions in order for this request to be accepted:") }}</h5>
+                                      {{ _(accept_conditions_text) | safe }}
```

Comment:
this is redundant I think because it will not mark the resolved text as translatable but rather the variable `accept_conditions_text` but maybe to be checked

Reply:
Saw the example here: https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/header.html#L33
But you're right, seems like we can't translate everything that user can put into that field